core/templates/login.php

```diff
@@ -4,14 +4,13 @@
 ?>
 
 <div id="login"></div>
+
 <?php if (!empty($_['alt_login'])) { ?>
-<form id="alternative-logins">
-	<fieldset>
-		<ul>
-			<?php foreach($_['alt_login'] as $login): ?>
-				<li><a class="button" href="<?php print_unescaped($login['href']); ?>" ><?php p($login['name']); ?></a></li>
-			<?php endforeach; ?>
-		</ul>
-	</fieldset>
-</form>
-<?php }
+    <div id="alternative-logins">
+        <?php foreach($_['alt_login'] as $login): ?>
+            <a class="button <?php p(isset($login['style']) ? $login['style'] : ''); ?>" aria-label="<?php p($login['label']); ?>" href="<?php print_unescaped($login['href']); ?>" >
```

Comment:
```suggestion
            <a class="button <?php p($login['style'] ?? ''); ?>" aria-label="<?php p($login['label'] ?? ''); ?>" href="<?php print_unescaped($login['href']); ?>" >
```

1) null coalesce makes it a bit more readable
2) label looks like a new attribute to me. we should use null coalesce here as well.

Reply:
1. yep, changed it
2. yep, its a new attribute. Added `$login['name']` as fallback

Reply:
Thx

Reply:
Again, `label` may be (and will be) undefined.